org-roam.el

```diff
@@ -1403,14 +1408,30 @@ file."
           (t
            'org-link))))
 
+;; light-weight function that is called during after-save-hook and only schedules the current
+;; orgmode file to be org-roam updated during the next idle slot
+(defun org-roam--update-db-when-idle (&optional file-path)
```

Comment:
maybe something like `org-roam--queue-file-for-update`. The comments above can go into the docstring.

Reply:
done, thank you!